decentraland/unity-renderer
# fix: insuficient resources crash when creating avatar for the first time

## What does this PR change?

`Fixes #2773 `

Fixes a loading freeze after you create your own avatar for the first time (guest or wallet).
This was caused by the emote list. Every emote was triggering the AvatarPreviewController to update many times in a row. Every update was being cancelled by throwing a `CanceledOperationException` catch by kernel and then flooding the analytics with it.

## How to test the changes?

1. Clear cookies so you enter as a new user
2. Open the developer console (F12 on chrome)
3. Enter as a guest
4. Start the avatar creation flow
5. Modify your avatar
6. Apply the changes
7. There should not be a flooding message related to `LoadProfile`
8. Set a name and accept the terms and conditions
9. The loading screen should not freeze at 0% and you should be able to enter the world
10. Inside the world go to the avatar editor, modify it and apply the changes. Everything should work as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
